create-regolith-term-profile

```diff
@@ -51,13 +51,16 @@ apply() {
     scrollbar="$(xrescat gnome.terminal.scrollbar never)" \
     useTransparentBg="$(xrescat gnome.terminal.use-transparent-background false)" \
     audibleBell="$(xrescat gnome.terminal.audible-bell false)" \
-    transparentBgPercent="$(xrescat gnome.terminal.background-transparency-percent 4)"
+    transparentBgPercent="$(xrescat gnome.terminal.background-transparency-percent 4)" \
+    blink="$(xrescat gnome.terminal.cursor-blink-mode off)"
 
   _write palette "['$color0', '$color1', '$color2', '$color3', '$color4', '$color5', '$color6', '$color7', '$color8', '$color9', '$color10', '$color11', '$color12', '$color13', '$color14', '$color15']"
   log 4 "Applied color color palette"
 
-  _write background-color "'$color0'"
-  _write foreground-color "'$color7'"
+  local background="$(xrescat gnome.terminal.background-color $color0)"
```

Comment:
I'm curious as to the use case for this change...can you explain?

Reply:
I had to set new locals for `foreground` and `background` because they fall back to `$color7` and `$color0` respectively, which are undefined when referenced in the context of the first local block. They are the same values as were being set previously though.